impeller/entity/shaders/gradients/fast_gradient.frag

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+precision mediump float;
+
+#include <impeller/color.glsl>
+#include <impeller/dithering.glsl>
+#include <impeller/types.glsl>
+
+uniform FragInfo {
+  float alpha;
+}
+frag_info;
+
+in vec4 v_color;
+
+out vec4 frag_color;
+
+void main() {
+  frag_color = IPPremultiply(v_color) * frag_info.alpha;
+  // mod operator is not supported in GLES 2.0
```

Comment:
Whats with this comment? Also, even though the operator is not supported, you can use the function.

Reply:
This shader would fail to load on GLES because the ordered dither uses a bunch of bit manipulation logic?

Reply:
Oh. Gotcha. That method has a mod in there somewhere. 

We should be able to fix it using the mod() function though. It's just the operator that's bad in ES.

Reply:
Nothing to do in this patch. Was just curious.

Reply:
ahh TIL!

Reply:
what an odd restriction